include/ck_tile/ops/epilogue/cshuffle_epilogue.hpp

```diff
@@ -13,7 +13,10 @@
 namespace ck_tile {
 
 template <typename T>
-concept HasDataType = requires { typename T::DataType; };
+concept HasDataType = requires
+{
+    typename T::DataType;
+};
```

Comment:
Avoid using C++20 features which breaks downstream projects.
```suggestion
template <typename T, typename = void>
constexpr bool HasDataType = false;
template <typename T>
constexpr bool HasDataType<T, typename T::DataType> = true;
```

Reply:
This is not my code change. This code is resulting from the clang formatting tool that we are using. If this tool suggests formatting changes then it appears in the commit as well.... which is resulting in unnecessary code diff results.

Reply:
looks like there are merge conflicts, so the merge base needs to be updated